### ISSUE TITLE
Pre-work Checklist: Developer: Luis Sotelo

### ISSUE THREAD

**Issue by LOSjr4** [hackforla/website#4070]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by LOSjr4** [comment]
Progress: Attended two meetings. Setting my system up until Docker.
Blockers: Docker installation had several issues. Also had a unrelated blocker keeping me from restarting my computer.
Availability: I uninstalled and in progress of reinstalling Docker. I have at least 6 hours over the next few days to continue through the list. 
ETA: this week starting today 3/14/2023 willing another blocker doesn't occur.

**Comment by LOSjr4** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
Maybe 13 hours over the past two weeks. I had several issues setting up docker, but I understand it better now. I attended two meetings 1 + 2 hours before I got to assigning myself my good first issue. First issue was a breeze though.

**Comment by LOSjr4** [comment]
First issue itself was a breeze, however I pushed it incorrectly. The github gui direction might have messed me up a bit, but I also rushed through it.
I made sure to go through the terminal instructions more carefully the second time.
On Tuesday I finished reading through everything else in this Pre-work Checklist. I will keep it handy as I progress through additional issues.

**Comment by ExperimentsInHonesty** [comment]
@LOSjr4 I noticed that you have not picked up a second issue.  It's possible when you looked for one, there was not one available.  

There is 1 available now [Good front end & Backend](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257).  

If you don't find one when you look again, go ahead and do another [good first issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257).  They are basically the same type of issue, we just need people to do two total so that we can make sure you dev environment is setup and you know how to do branching effectively.

**Comment by LOSjr4** [comment]
Progress: Didn't find appropriate work on 03/23. On 3/27 I reviewed an issue and requested a change to that issue, and assigned myself a new issue. Today I saw the reviewed issue was revised so I checked and approved it. Then I completed my good second issue and made a PR.
Blockers: Just lack of issues that day I was open to work. Over the weeks the types of issues I can comfortably review will widen. 
Availability: Mon, Tues, Thurs 9AM-1PM. Kid's Spring Break is next week will affect availability.
ETA: getting to a "Large issue" likely mid April

**Comment by jdingeman** [comment]
Hi @LOSjr4 - please provide us with an update on this issue when you can, thanks!

**Comment by LOSjr4** [comment]
1. Progress: Last week I completed a Medium issue and this week I was excited to take a Large issue, but instead I took another Medium issue because the Large issues were difficult to grasp at my current understanding of HFLA inner workings. I want to look for opportunities to learn more about HFLA over the next few meetings to help to easier understand my future Large issue. Also I've gotten used to issues being completed in a day or two, so now I'm starting to accept the idea that some Large issues have multiple steps that could take a week or so.
2. Blockers: No major blockers yet.
3. Availability: Generally M,T,TH 9AM-1PM
4. ETA: Hopefully next week I'm better prepared for a Large issue.

**Comment by LOSjr4** [comment]
Progress: My last medium issue was new and exciting involving Ruby, Bundle and Jekyll to explore an alternative to Docker. 
Yesterday and today I keeping checking in for Large issues but nothing new has popped up. PR are also very limited this week.
At Tuesday's meeting I offered to collaborate with Niva R. on her issue. So far I've shared my previous issue findings to her Slack as it may be loosely related.
Blockers: Only lack of new work at the moment.
Availability: Generally M,T,TH 9AM-1PM
ETA: Still waiting to tackle a Large issue.

**Comment by LOSjr4** [comment]
Progress: I'm currently working on my first Large issue (#4555)!
Roslyn Wythe, Justin Dingeman and myself are coordinating a meetup for a recorded demonstration of one of my previous research issues (#4409).
Blockers: None. I'm happy to have a large issue.
Availability: Generally M,T,TH 9AM-1PM
ETA: 5/8/23

**Comment by LOSjr4** [comment]
Progress: Clarified my Large issue with dev leads. Back to working on it this week.
Blockers: According to my issue's task list, I was to request HackforLA Slack sign in credentials from Bonnie, but now focusing on investigating the feasibility before attempting to implement the change.
Availability: Weekdays 9AM-1PM
ETA: This week

**Comment by LOSjr4** [comment]
Progress: I'm still working on Large issue #4555. I received a "slackinvite project" email account to create new Slack and UiPath accounts and set up 2FA with 1Password using the given guide. Next I need to build the automation.
Blockers: When I was about to start building the automation I realized I needed to clarify certain steps toward the end (and UiPath requires every single step to work), so I'm conversing with the issue creator, Christina Or. Otherwise I'll bring it up on Sunday's meeting.
Availability: weekdays 9AM-1PM
ETA: hopefully early next week

**Comment by LOSjr4** [comment]
Progress: Still working through issue https://github.com/hackforla/website/issues/4555
 and reviewing at least two PRs each week. I clarified the steps I was missing before. Programming the automation has been trickier than expected.
Blockers: A popup needs to be cleared during the UiPath automation. According to forums, it should be possible to select a popup and perform a click, but so far I haven't been able to get UiPath to acknowledged that popup.
Availability: weekdays 9AM-1PM
ETA: this week

**Comment by LOSjr4** [comment]
Last few weeks were busy, but I'm still working on my large issue using UiPath. I'd like to complete it soon, but small issues during automation require me to troubleshoot and learn more about UiPath.
I'm enjoying the learning process.

**Comment by LOSjr4** [comment]
Through a lot of trial and error, I could almost see the end of issue #4555. Maybe two more weeks to test a timed automation and finalizing the automation steps.

**Comment by LOSjr4** [comment]
I will be away from 7/13-7/20. I will continue #4555 when I return.

**Comment by LOSjr4** [comment]
I have returned from vacation. I will attend 7/23 meeting to attempt to proceed with an UiPath pair programming.

**Comment by roslynwythe** [comment]
Hi @LOSjr4 I'm closing this issue as per our discussion, but your work was excellent and we enjoyed working with you, and we hope that you return in the future.  In that case, you are welcome to reopen this issue, or if you lack rights to do that, simply send a Slack message to a dev lead.  Thank you for your contributions to Hack for LA.

**Comment by HackforLABot** [comment]
## Activity Log: LOSjr4
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- LOSjr4 opened: https://github.com/hackforla/website/issues/4070 at 02/28/2023, 08:16 PM PST
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4070 at 02/28/2023, 08:16 PM PST
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1468577573 at 03/14/2023, 11:00 AM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/3950#issuecomment-1445356749 at 03/16/2023, 01:01 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/3950#issuecomment-1472693332 at 03/16/2023, 01:27 PM PDT
- LOSjr4 closed by PR 4199: https://github.com/hackforla/website/issues/3950#event-8771860945 at 03/16/2023, 01:32 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1472713610 at 03/16/2023, 01:43 PM PDT
- LOSjr4 unassigned: https://github.com/hackforla/website/issues/3950#issuecomment-1473024302 at 03/16/2023, 07:36 PM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/3948 at 03/16/2023, 08:04 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/3948#issuecomment-1473052444 at 03/16/2023, 08:08 PM PDT
- LOSjr4 opened: https://github.com/hackforla/website/pull/4203 at 03/17/2023, 02:19 PM PDT
- LOSjr4 PR was merged: https://github.com/hackforla/website/pull/4203#event-8789910921 at 03/19/2023, 08:50 PM PDT
- LOSjr4 issue closed by PR 4203: https://github.com/hackforla/website/issues/3948#event-8789911053 at 03/19/2023, 08:50 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1481998627 at 03/23/2023, 03:30 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4290#issuecomment-1485478697 at 03/27/2023, 09:47 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1359994177 at 03/27/2023, 04:18 PM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4284 at 03/27/2023, 04:22 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4284#issuecomment-1485992444 at 03/27/2023, 04:26 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1361738253 at 03/28/2023, 11:59 AM PDT
- LOSjr4 opened: https://github.com/hackforla/website/pull/4306 at 03/28/2023, 02:32 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1487640587 at 03/28/2023, 02:52 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4306#issuecomment-1487918298 at 03/28/2023, 09:09 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4306#issuecomment-1490567816 at 03/30/2023, 09:11 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4306#issuecomment-1490625528 at 03/30/2023, 09:54 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4346#issuecomment-1490668024 at 03/30/2023, 10:27 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4346#pullrequestreview-1365622433 at 03/30/2023, 10:44 AM PDT
- LOSjr4 PR was merged: https://github.com/hackforla/website/pull/4306#event-8893133084 at 03/30/2023, 06:15 PM PDT
- LOSjr4 issue closed by PR 4306: https://github.com/hackforla/website/issues/4284#event-8893133183 at 03/30/2023, 06:15 PM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4293 at 04/04/2023, 12:02 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4293#issuecomment-1496465607 at 04/04/2023, 12:08 PM PDT
- LOSjr4 opened: https://github.com/hackforla/website/pull/4408 at 04/04/2023, 01:03 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4406#issuecomment-1496545354 at 04/04/2023, 01:13 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4406#pullrequestreview-1371864623 at 04/04/2023, 03:15 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4404#issuecomment-1496686653 at 04/04/2023, 03:37 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4404#pullrequestreview-1371893404 at 04/04/2023, 03:51 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4406#pullrequestreview-1371900162 at 04/04/2023, 03:59 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4406#issuecomment-1496707676 at 04/04/2023, 04:07 PM PDT
- LOSjr4 PR was merged: https://github.com/hackforla/website/pull/4408#event-8940852084 at 04/05/2023, 06:37 PM PDT
- LOSjr4 issue closed by PR 4408: https://github.com/hackforla/website/issues/4293#event-8940852156 at 04/05/2023, 06:38 PM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4392 at 04/11/2023, 07:47 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4392#issuecomment-1503516381 at 04/11/2023, 07:49 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4392#issuecomment-1505945218 at 04/12/2023, 02:06 PM PDT
- LOSjr4 opened: https://github.com/hackforla/website/pull/4501 at 04/13/2023, 10:18 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4500#issuecomment-1507338951 at 04/13/2023, 10:23 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4500#pullrequestreview-1383919972 at 04/13/2023, 10:40 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4497#issuecomment-1507370344 at 04/13/2023, 10:43 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4497#pullrequestreview-1383944712 at 04/13/2023, 10:54 AM PDT
- LOSjr4 PR was merged: https://github.com/hackforla/website/pull/4501#event-9015928417 at 04/16/2023, 12:16 AM PDT
- LOSjr4 issue closed by PR 4501: https://github.com/hackforla/website/issues/4392#event-9015928462 at 04/16/2023, 12:16 AM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4409 at 04/17/2023, 10:18 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1511776805 at 04/17/2023, 10:20 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1513479912 at 04/18/2023, 09:38 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1513806791 at 04/18/2023, 02:18 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1515022571 at 04/19/2023, 09:21 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1515041488 at 04/19/2023, 09:34 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4497#pullrequestreview-1392486081 at 04/19/2023, 09:44 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1515523785 at 04/19/2023, 04:58 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4495#issuecomment-1516831169 at 04/20/2023, 12:23 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4495#pullrequestreview-1394645054 at 04/20/2023, 12:36 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4495#issuecomment-1516855884 at 04/20/2023, 12:42 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1524139042 at 04/26/2023, 04:00 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4573#issuecomment-1525933455 at 04/27/2023, 08:47 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4573#pullrequestreview-1404485724 at 04/27/2023, 09:44 AM PDT
- LOSjr4 assigned: https://github.com/hackforla/website/issues/4555 at 05/01/2023, 09:55 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1529950977 at 05/01/2023, 09:59 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4573#pullrequestreview-1407900964 at 05/01/2023, 11:02 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1533739380 at 05/03/2023, 01:58 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1535205291 at 05/04/2023, 11:13 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4613#issuecomment-1540582195 at 05/09/2023, 10:28 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4613#pullrequestreview-1419347307 at 05/09/2023, 12:41 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4600#issuecomment-1541196567 at 05/09/2023, 07:22 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4600#pullrequestreview-1423189916 at 05/11/2023, 11:22 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4600#pullrequestreview-1424859785 at 05/12/2023, 09:53 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1546030465 at 05/12/2023, 09:58 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1548515675 at 05/15/2023, 01:16 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4691#pullrequestreview-1433294234 at 05/18/2023, 12:05 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1555289692 at 05/19/2023, 02:46 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4688#issuecomment-1556235821 at 05/21/2023, 10:36 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4688#pullrequestreview-1437413199 at 05/22/2023, 01:18 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1560038875 at 05/23/2023, 12:52 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4731#issuecomment-1563622068 at 05/25/2023, 04:17 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4730#pullrequestreview-1444778690 at 05/25/2023, 04:38 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1564652762 at 05/26/2023, 09:41 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1564661338 at 05/26/2023, 09:49 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4732#pullrequestreview-1446679462 at 05/26/2023, 10:37 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4745#issuecomment-1566203053 at 05/28/2023, 10:13 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4748#issuecomment-1566203487 at 05/28/2023, 10:14 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1566223041 at 05/28/2023, 11:27 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4732#pullrequestreview-1451327515 at 05/30/2023, 09:11 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4745#pullrequestreview-1451343294 at 05/30/2023, 09:20 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4748#pullrequestreview-1451366232 at 05/30/2023, 09:33 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4409#issuecomment-1570590607 at 05/31/2023, 09:58 AM PDT
- LOSjr4 issue closed as completed: https://github.com/hackforla/website/issues/4409#event-9425575069 at 06/04/2023, 11:47 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4789#pullrequestreview-1468083682 at 06/07/2023, 09:07 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4783#pullrequestreview-1468199180 at 06/07/2023, 10:06 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1581281044 at 06/07/2023, 11:04 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1589620307 at 06/13/2023, 09:11 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4783#pullrequestreview-1477596409 at 06/13/2023, 09:17 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4843#issuecomment-1593968455 at 06/15/2023, 06:51 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4843#pullrequestreview-1482641586 at 06/15/2023, 07:48 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1595211639 at 06/16/2023, 12:50 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4837#issuecomment-1603332729 at 06/22/2023, 02:20 PM PDT
- LOSjr4 opened: https://github.com/hackforla/website/pull/4880 at 06/24/2023, 09:48 AM PDT
- LOSjr4 closed: https://github.com/hackforla/website/pull/4880#event-9627315869 at 06/24/2023, 09:49 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4837#pullrequestreview-1496763662 at 06/24/2023, 10:27 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4875#pullrequestreview-1496768234 at 06/24/2023, 11:00 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1612314738 at 06/28/2023, 06:47 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1612322551 at 06/28/2023, 07:00 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1612352861 at 06/28/2023, 07:35 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4897#pullrequestreview-1504504939 at 06/28/2023, 08:35 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4890#pullrequestreview-1504531402 at 06/28/2023, 09:26 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1612428288 at 06/28/2023, 09:51 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4911#pullrequestreview-1515053806 at 07/05/2023, 10:55 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/4925#issuecomment-1627776002 at 07/09/2023, 10:15 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/4925#pullrequestreview-1526877767 at 07/12/2023, 10:39 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1632950142 at 07/12/2023, 10:42 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1632979424 at 07/12/2023, 11:01 AM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/5050#pullrequestreview-1542176301 at 07/22/2023, 07:25 PM PDT
- LOSjr4 submitted review: https://github.com/hackforla/website/pull/5044#pullrequestreview-1542177113 at 07/22/2023, 07:42 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/pull/5050#issuecomment-1646730252 at 07/22/2023, 07:44 PM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4070#issuecomment-1646731282 at 07/22/2023, 07:49 PM PDT
- LOSjr4 issue closed as completed: https://github.com/hackforla/website/issues/4070#event-9991372030 at 08/02/2023, 11:54 AM PDT
- LOSjr4 commented on: https://github.com/hackforla/website/issues/4555#issuecomment-1664393234 at 08/03/2023, 10:50 AM PDT
- LOSjr4 unassigned: https://github.com/hackforla/website/issues/4555 at 08/03/2023, 10:51 AM PDT
